resources/post-upgrade.sh

```diff
@@ -120,20 +114,17 @@ if [[ ${FROM_VERSION} == *"5.6.7"* ]] && [[ ${TO_VERSION} == *"6.7."* ]]; then
 
     echo "Waiting for SonarQube to get healthy (max. ${WAIT_TIMEOUT} seconds)..."
     # default admin credentials (admin, admin) are used
-    wait_for_sonar_to_get_healthy ${WAIT_TIMEOUT} admin admin ${CURL_LOG_LEVEL}
+    wait_for_sonar_to_get_healthy ${WAIT_TIMEOUT} "${TEMPORARY_ADMIN_USER}" "${PW}" ${CURL_LOG_LEVEL}
 
-    reinstall_plugins
+    reinstall_plugins "${TEMPORARY_ADMIN_USER}" "${PW}"
 
-    # clear install_plugins key
-    doguctl config install_plugins "" >> /dev/null
+    doguctl config --remove install_plugins
   fi
 
-  # Do everything that needs to be done to get into a state that is equal to a successful first start
-  create_dogu_admin_and_deactivate_default_admin ${CURL_LOG_LEVEL}
-  set_successful_first_start_flag
```

Comment:
Why are these lines not necessary any more? 🤔

Reply:
We needed these lines in cases where we upgraded from an old version, which didn't set the "successful_first_start_flag" nor created the dogu admin. We only execute this post_upgrade.sh if we come from a SonarQube 6, which already does this.